start-site/src/main/resources/application.yml

```diff
@@ -1274,7 +1274,7 @@ initializr:
           artifactId: spring-cloud-stream
     - name: VMware Tanzu Application Service
       bom: spring-cloud-services
-      compatibilityRange: "[2.0.0.RELEASE,2.5.0-M1)"
+      compatibilityRange: "[2.0.0.RELEASE,2.6.0-RC1)"
```

Comment:
I don't understand the range. Did you mean `2.6.0-M1` ?

Reply:
It was meant to be any version of 2.5.x (but not 2.6).  So I used the first RC version of 2.6 (exclusive).

Reply:
`2.6.0-RC1` is not the first version in the 2.6.x line so that change would mean any milestone of 2.6.x would be included (`2.6.0-M1`, `2.6.0.M2`, etc. up to `2.6.0-RC1` excluded).

Reply:
sorry, my bad.